src/main/java/frc/robot/BuildConstants.java

```diff
@@ -7,12 +7,12 @@ public final class BuildConstants {
   public static final String MAVEN_GROUP = "";
```

Comment:
This shouldn't be checked into git

Reply:
Fixed?

Reply:
It seems that Git will not build without the Build Constaints file.

Reply:
Yeah, so you can delete it, and then run build.  It will recreate the BuildConstants.java, and that's fine.  That will get sent to the robot.  But we don't want it stored in git

Reply:
you might be right

Reply:
fixed